/serial_mapping:
  ros__parameters:
    dds_baudrate: 2000000
    gcs_ip: 192.168.1.10  # 127.0.0.1 or 192.168.1.10

/amc_1:
  ros__parameters:
    test_phase: formation
    lasting_time: 30
    mc_hgt_ki: 0.05
    mc_hgt_kp: 0.5
    mc_hgt_sp: 5.0
    mc_yaw_sp: 0.0
    origin_lat: 24.43758
    origin_lon: 118.09782

/amc_2:
  ros__parameters:
    test_phase: formation
    lasting_time: 30
    mc_hgt_ki: 0.05
    mc_hgt_kp: 0.5
    mc_hgt_sp: 5.0
    mc_yaw_sp: 0.0
    origin_lat: 24.43758
    origin_lon: 118.09782

/amc_3:
  ros__parameters:
    test_phase: formation
    lasting_time: 30
    mc_hgt_ki: 0.05
    mc_hgt_kp: 0.5
    mc_hgt_sp: 5.0
    mc_yaw_sp: 0.0
    origin_lat: 24.43758
    origin_lon: 118.09782
    
